specificatie/openapi.yaml

```diff
@@ -1111,8 +1126,8 @@ components:
       description: |
                     Gegevens over de voltrekking van het huwelijk of het aangaan van het geregistreerd partnerschap.
                     * **datum** : De datum waarop het huwelijk is voltrokken of het partnerschap is aangegaan.
-                    * **land** : Het land waar het huwelijk is voltrokken of het partnerschap is aangegaan.
-                    * **plaats** : De gemeente waar het huwelijk is voltrokken of het partnerschap is aangegaan. Voor een plaats buiten Nederland bevat het antwoord een buitenlandse plaatsnaam of aanduiding.
+                    * **land** : Het land waar het huwelijk is voltrokken of het partnerschap is aangegaan,  wordt gevuld op basis van de waarden die voorkomen in de tabel 'land' uit de Haal-Centraal-BRP-tabellen-bevragen API.
+                    * **plaats** : De gemeente waar het huwelijk is voltrokken of het partnerschap is aangegaan. Voor een plaats buiten Nederland bevat het antwoord een buitenlandse plaatsnaam of aanduiding. Wordt gevuld op basis van de waarden die voorkomen in de tabel 'woonplaats' uit de Haal-Centraal-BRP-tabellen-bevragen API.
```

Comment:
kan je de laatste zin niet beter achter de eerste zetten, want dit heeft alleen betrekking op een Nederlandse gemeente. De tweede zin (over buitenlandse plaats of locatie) komt daar dan achter.